opentelemetry/src/sdk/trace/span_processor.rs

```diff
@@ -125,13 +130,17 @@ impl SpanProcessor for SimpleSpanProcessor {
         }
     }
 
-    fn force_flush(&self) {
+    fn force_flush(&self) -> ExportResult {
         // Ignored since all spans in Simple Processor will be exported as they ended.
+        Ok(())
     }
 
-    fn shutdown(&mut self) {
+    fn shutdown(&mut self) -> ExportResult {
```

Comment:
Do we need to call `force_flush` in `shutdown`? From my understanding, `shutdown` need to do some cleanup before shutdown.

Reply:
You are right. The spec says shutdown must contains force_flush. But in `SimpleSpanProcessor`'s force_flush does nothing so there is really no point to call it.